examples/ssoadmin/customermanagedpolicyattachment.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: ssoadmin.aws.upbound.io/v1beta1
+kind: CustomerManagedPolicyAttachment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ssoadmin/v1beta1/customermanagedpolicyattachment
```

Could you please add a manual intervention like below?

```suggestion
    meta.upbound.io/example-id: ssoadmin/v1beta1/customermanagedpolicyattachment
    upjet.upbound.io/manual-intervention: "This resource depends on PermissionSet. This resource is skipping because the PermissionSet resource was skipped."
```

---

I think there's a missing piece in my understanding of what these annotations are used for.

In my mental model, the goal of this annotation is to avoid creating managed resources that are in a state which will never become ready, because there is some additional information needed to create them. The user running the test needs to manually create them after the test has started, and then they will become ready, and the test will finish and pass. In that model, it seems like it would be better to _not_ have a manual intervention annotation on this resource, because if it's applied as is, it will simply wait for the permissionset to be (manually) created and ready, thanks to the selector, and then it will become ready without manual intervention needed. 

If we put the manual intervention annotation on, it just makes more manual work for the user running the test.

What am I overlooking/misunderstanding?

---

You are correct when we consider it from your mental model, but the simple purpose of placing manual intervention is skipping the automatic test. The aim is to provide a brief explanation for the reason behind the skip, ensuring that the person triggering the test is informed.

In this case, a person who triggers an automatic test(uptest) on the `CustomerManagedPolicyAttachment` resource can see why it was skipped with this annotations `upjet.upbound.io/manual-intervention: "This resource depends on PermissionSet. This resource is skipping because the PermissionSet resource was skipped."`

---

ok, I'll add annotations to all the resources which depend on resources with manual intervention annotations. I think that will amount to all of the resources in all the examples except the IAM policies.